Cloudinary.Test/AsyncMethodsTest.cs

```diff
@@ -159,6 +160,32 @@ public void TestDeleteAsync()
             Assert.IsTrue(String.IsNullOrEmpty(resource.PublicId));
         }
 
+        [Test]
+        public void TestDeleteTransformAsync()
+        {
+            // should allow deleting named transformation
+            string transformationName = GetUniqueTransformationName();
+
+            CreateTransformParams create = new CreateTransformParams()
+            {
+                Name = transformationName,
+                Transform = m_simpleTransformation
+            };
+
+            TransformResult createResult = m_cloudinary.CreateTransformAsync(create).Result;
+
+            Assert.AreEqual("created", createResult.Message);
```

Comment:
It would be much more elegant to extract transformation creation functionality to a separate function, or move all transformation related tests to a separate class and move this code to [setUp] method.

Make sure that transformation is created as intended.